.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+
+# One line description
+
+Please include a paragraph here describing what this PR does:
+what kind of change (refactoring, bugfix, feature), what is changed
+(short summary of new features or fix).
+
+If bugs are addressed in this PR, include the issue numbers.
+
+# Who benefits?
+
+Axom team, "LLNL customers" if you can't mention in detail,
+specific LLNL projects if you can, or outside feature request.
```

Comment:
This section and paragraph is here to help justify the PR. Ideally, each change should be clearly motivated by a feature request. Even though this is far from the real situation, a clear statement of need will help improve our code's quality and review process.

Reply:
I think I understand your intent here. However, my fear is that we will end up defaulting this to `LLNL customers` and not pay too much attention to it, which then defeats its purpose.  

Also, the separation between who benefits, e.g., Axom developers vs. Axom users is not always clear. Put simply, each PR benefits the Axom community and that includes both users and developers of Axom.

Moreover, from the perspective of an external contributor perspective, this section of the PR template may be a bit awkward.  

Another way to do this is to use appropriate Github labels instead as needed. That would also allow us to easily search/filter/find things as well.

Reply:
I changed the template to suggest user requests for features in the initial list.  I still think listing "who wanted this" right on the PR summary is a good way to keep focused on why Axom is here: to benefit other codes.  Please look and comment on the updated template.